idr/.snapshots/TestRemoveNodeAndSubTree-remove_a_node_who_is_its_parents_first_child_but_not_the_last

```diff
@@ -0,0 +1,82 @@
+{
+	"Children": [
+		{
+			"Children": [
+				{
+					"Children": [
+						{
+							"Children": null,
+							"Data": "data 21",
+							"FirstChild": "(nil)",
```

Comment:
How do people understand the differences between `null` and `"(nil)"`?

Reply:
changed to using *string and null